carma/launch/environment.launch

```diff
@@ -101,6 +103,8 @@
   </group>
 
 <!-- vision_beyond_track -->
+  <remap to="detection/image_detector/objects" from="/environment/vision_beyond_track_01/detection/image_detector/objects"/>
```

Comment:
is the leaving /environment/ needed for the remap here since that is the namespace of this file?

Reply:
removed leading /environment